csrankings-4.csv

```diff
@@ -1621,7 +1621,7 @@ Kartikeya Mayaram,Oregon State University,http://eecs.oregonstate.edu/people/may
 Karu Sankaralingam,University of Wisconsin - Madison,http://pages.cs.wisc.edu/~karu,O0W9jEQAAAAJ
 Kashif Munir,King Abdulaziz University,http://fcit2.kau.edu.sa:9002/reports/rwservlet?cmdkey=FCVKEY+p_fac_id=00062561,BHPpU8oAAAAJ
 Kasim Selçuk Candan,Arizona State University,http://aria.asu.edu/candan,keO-0s0AAAAJ
-Kasım Sinan Yildirim,University of Trento,http://https://sinanyil81.github.io,LXUvnL0AAAAJ
+Kasim Sinan Yildirim,University of Trento,http://https://sinanyil81.github.io,LXUvnL0AAAAJ

```

Comment:
Please fix the URL (starts with "http://https://" - pick one :) ).

Reply:
I have just done it, sorry:)